app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -27,17 +29,25 @@ class Uploader extends \Magento\Backend\Block\Widget
      */
     protected $_fileSizeService;
 
+    /**
+     * @var \Magento\Framework\Image\Adapter\ConfigInterface
+     */
+    protected $imageConfig;
+
     /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Framework\File\Size $fileSize
+     * @param \Magento\Framework\Image\Adapter\ConfigInterface $imageConfig
      * @param array $data
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Framework\File\Size $fileSize,
+        \Magento\Framework\Image\Adapter\ConfigInterface $imageConfig,
         array $data = []
     ) {
         $this->_fileSizeService = $fileSize;
+        $this->imageConfig = $imageConfig;
         parent::__construct($context, $data);
```

Comment:
Please check https://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#php on how to add new dependency properly, always try to make parent constructor called first.